services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/SwitchFeatures.java

```diff
@@ -74,12 +78,16 @@
     @Property(name = "support_vxlan_vni_match")
     private boolean supportVxlanVniMatch;
 
+    @Property(name = "supported_transit_encapsulation")
+    private Set<FlowEncapsulationType> supportedTransitEncapsulation;
+
     @Builder(toBuilder = true)
     public SwitchFeatures(Switch switchObj, boolean supportBfd, boolean supportVxlanPushPop,
-                          boolean supportVxlanVniMatch) {
+                          boolean supportVxlanVniMatch, Set<FlowEncapsulationType> supportedTransitEncapsulation) {
```

Comment:
It's good idea.

Reply:
Just curious - why not "supportedFlowEncapsulation" ?

Reply:
wanted to focus on the fact it's about transit rules mostly